### DISCUSSION TITLE
Integrate FireEye Helix

### DISCUSSION THREAD

**Discussion by RoyTM99** [Security-Onion-Solutions/securityonion#7057] [Unsupported Versions] [answered]
Good Day Team,

The reason for this request is to ask for your kind support in order to know if there is any documentation or instructions to follow about how to integrate the FireEye Helix cloud solution to the SIEM. As far as I have found there is an option to integrate a security onion sensor to FireEye according to the reference link below; but what I am looking for now is to integrate FireEye as another LogSource to SIEM (such as a Firewall or Antivirus). Is there any alternative to make this possible? 

Reference
https://www.fireeye.com/blog/products-and-services/2019/11/expanded-network-visibility-in-fireeye-helix-with-security-onion-solutions.html

Security Onion Version
2.3.90

I remain attentive to your comments.
Kind Regards
Roy T.

**Comment by dougburks** [comment, marked as answer]
If it can send logs, then you may be able to develop an ingest parser:
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing

**Comment by Phillip-Samson** [comment]
@RoyTM99 
Did you happen to create an ingest parser for FireEye.
We are trying to do the same thing. at my company

**Reply by RoyTM99** [reply]
Hi @Bal33p ,

I comment that for the moment we did not go deeper into the issue, for various reasons we did not follow up on this integration, but just a few days ago we took it up again and we found that our FireEye on Cloud does not send logs by Syslog, only has the option of HTTP and SMTP of which we have some doubts, we asked the vendor if there are other options for sending logs but so far we have no response, have you already been able to send FireEye logs in any way, maybe some API or HTTP is the option? 

Kind Regards
